modules/insights-operator-register-unconnected-cluster.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc
+// * sd_support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc
+
+
+:_content-type: PROCEDURE
+[id="insights-operator-register-unconnected-cluster_{context}"]
+= Registering your unconnected cluster
+
+Register your unconnected {product-title} cluster on the {hybrid-console} so that your cluster is not impacted by the consequences listed in the section named "Consequences of disabling remote health reporting".
+
+[IMPORTANT]
+====
+By registering your unconnected cluster, you can continue to report your subscription usage to Red Hat. In turn, Red Hat can return accurate usage and capacity trends associated with your subscription, so that you can use the returned information to better organize subscription allocations across all of your resources.
+==== 
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as `cluster-admin`.
+* You can log in to the {hybrid-console}.
+
+.Procedure
+. Go to the link:https://console.redhat.com/openshift/register[*Register disconnected cluster*] web page on the {hybrid-console}. 
+
+. Optional: To access the *Register disconnected cluster* web page from the home page of the {hybrid-console}, select the *Clusters* navigation menu item and then select the *Register cluster* button.
+
+. Enter your cluster's details in the provided fields on the *Register disconnected cluster* page. 
+
+. From the *Subscription settings* section of the page, select the radio buttons that apply to your Red Hat subscription offering. 
```

Comment:
Per the ISG, avoid the use of _radio buttons_, refer to the control by the label. 
```suggestion
. From the *Subscription settings* section of the page, select the subscription settings that apply to your Red Hat subscription offering. 
```